WordImpress/Give
# Issues 2037

## Description
PR for issues Id: #2037 

## How Has This Been Tested?
Manually Testing
See the Video how I have tested it `http://screencast-o-matic.com/watch/cbQFqEIYFI` ( update )

## Types of changes
New feature (non-breaking change which adds functionality)

## Checklist:
